core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/DataFrameGet.kt

```diff
@@ -58,6 +59,36 @@ public fun <T> ColumnsContainer<T>.getFrameColumn(columnName: String): FrameColu
 public fun <T> ColumnsContainer<T>.getColumnGroup(columnPath: ColumnPath): ColumnGroup<*> =
     get(columnPath).asColumnGroup()
 
+/**
+ * Utility property to access scope with only dataframe column properties for code completion,
+ * filtering out DataFrame API.
+ *
+ * It's a quick way to check that code generation in notebooks or compiler plugin
+ * worked as expected or find columns you're interested in.
+ *
+ * In notebooks:
+ * ```
+ * val df = DataFrame.read("file.csv")
+ * ==== next code cell
+ * df. // column properties are mixed together with methods, not easy to find unless you already know names
+ * df.properties(). // easy to overview available columns
+ * ```
+ * In compiler plugin:
+ * ```
+ * val df = @Import DataFrame.read("file.csv")
+ * df.properties().
+ * ```
+ * By design, it can't be used in columns selection DSL. Because "properties()" is optional - for convenient overview.
```

Comment:
hmm I'd leave out this bit. Users can figure out for themselves this doesn't work and all your examples are calling upon a `DataFrame` instance anyway